React/Project structure.md

```diff
@@ -79,6 +266,8 @@ Example:
     └── WelcomeCard.elements.ts
 ```
 
+> Moving things to`.elements.tsx` should be the last step in the development process and it should only be used for organizational purposes, i.e. when the main component becomes cluttered and unreadable.
```

Comment:
Not sure about this, I think that every components should have`.elements.tsx` instead of having styles in main file. Almost every component will have some kind of wrapper or something.